langgraph_bot/main.py

```
@@ -1,43 +1,49 @@
+            result = mjorgraph.invoke(initial_state)
 
             # Return clean structured output with only relevant fields
             clean_post = {
                 "title": result.get("title") or post.get("title"),
                 "summary": result.get("summary"),
                 "description": result.get("description"),
                 "source": post.get("source_name"),
+                "slug":result.get("slug") or ""
             }
+            time.sleep(12)
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)